utils/src/unit-test/java/uk/gov/moj/sdt/utils/web/filter/ContextCleanupFilterTest.java

```diff
@@ -88,33 +99,22 @@ public void setUp() {
      * @throws IOException      the exception thrown on IO problems
      */
     @Test
-    public void doFilterTest() throws IOException, ServletException {
+    void doFilterTest() throws IOException, ServletException {
         // Add code to set something in SdtContext.
         SdtContext.getContext().setRawInXml("TestXml");
 
-        Assert.assertNotNull(SdtContext.getContext().getRawInXml());
+        assertNotNull(SdtContext.getContext().getRawInXml());
 
         contextCleanupFilter = new ContextCleanupFilter();
 
         mockFilterChain.doFilter(mockServletRequest, mockServletResponse);
 
-        EasyMock.expectLastCall();
-
-        EasyMock.replay(mockServletRequest);
-        EasyMock.replay(mockServletResponse);
-        EasyMock.replay(mockFilterChain);
-
         contextCleanupFilter.doFilter(mockServletRequest, mockServletResponse, mockFilterChain);
 

```

Comment:
Previous version of test verified mocked objects - should this do the same?

Reply:
reintroduced